src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsremandandsentencingapi/controller/dto/CreateRecall.kt

```diff
@@ -0,0 +1,12 @@
+package uk.gov.justice.digital.hmpps.hmppsremandandsentencingapi.controller.dto
+
+import uk.gov.justice.digital.hmpps.hmppsremandandsentencingapi.jpa.enum.RecallType
+import java.time.LocalDate
+
+data class CreateRecall(
+  val prisonerId: String,
+  val recallDate: LocalDate,
+  val returnToCustodyDate: LocalDate,
+  val recallType: RecallType,
+  val createdByUsername: String,
```

Comment:
The caller to this endpoint uses the system client token, so passed in username in the body